source/views/menus/components/fancy-menu.js

```diff
@@ -146,6 +146,7 @@ class FancyMenuView extends React.Component {
     )
 
     let messageView = null
+    let filtersEnabled = filters.some(f => f.enabled)
```

Comment:
Can we rename this variable to "anyFiltersEnabled" instead? That seems like it would go better with the accompanying "specialsFilterEnabled" variable.